src/dxvk/dxvk_device.cpp

```diff
@@ -274,6 +275,7 @@ namespace dxvk {
           DxvkSubmitStatus*         status) {
     DxvkSubmitInfo submitInfo = { };
     submitInfo.cmdList = commandList;
+    submitInfo.frameId = m_latencyMarkers.render;
```

Comment:
Note that resource initialization (zeroing) submissions can happen independently from the immediate context or CS thread. `submitCommandList` is used by every submission including init submissions, and this can cause slightly awkward consequences such as submission tied with a particular frame ID after present.

It might be better to exclude InitContext from low latency tracking completely to avoid edge cases.

Reply:
Updated to disable passing frame ids at vkQueueSubmit time for work submitted from the initializer context.